core/LocOffsets.h

```diff
@@ -9,7 +9,7 @@ class GlobalState;
 class Context;
 class MutableContext;
 
-constexpr int INVALID_POS_LOC = 0xfffffff;
+constexpr uint32_t INVALID_POS_LOC = UINT32_MAX;
```

Comment:
Using those upper four bits seems pretty reasonable to me. Did we have plans for them at one point that we abandoned?

Reply:
I don't know, I can't think that this was anything other than a typo.

Reply:
File begin and end positions used to fit entirely within 2^24 bytes, because a `Loc` was

```
2 bytes: FileRef id
3 bytes: beginPos
3 bytes: endPos
```

and that stopped when we broke apart `Loc` into `LocOffsets`.